assets/src/scss/base/_mixins.scss

```diff
@@ -55,10 +55,10 @@
 }
 
 // Margins and padding. Use to consistently apply gutters
-// use like this: @include padding($left: $n15, $right: $n15); wich will be compiled as padding-left: 15px; padding-right: 15px;
+// use like this: @include padding($left: $sp-2, $right: $sp-2); wich will be compiled as padding-left: 15px; padding-right: 15px;
 
 // To use as negative numbers interpolate these as follows...
-// @include margin($top: -#{$n10}, $right: $n20, $bottom: $n10, $left: 0); which will be compiled as margin: -10, 20px 10px 0px;
+// @include margin($top: -#{$sp-1}, $right: $sp-2x, $bottom: $sp-1, $left: 0); which will be compiled as margin: -10, 20px 10px 0px;
```

Comment:
The comment here needs to be updated since it still uses multiples of 10 🙃

Reply:
(same on the line above btw, which uses multiples of 5)

Reply:
Good catch :grin: Comment updated.